### PR TITLE
WT-12398 Add backup related statistics.

### PR DESCRIPTION
**Pull request by sueloverso**
@kommiharibabu I added you because you opened the other duplicate ticket about backup stats.

### CODE REVIEW

**Comment by sueloverso** [comment]
I converted this to a draft because I want to consider per-table statistics in addition to connection level stats. So there may be changes coming to this PR.

**Comment by sueloverso** [comment]
@ddanderson and @kommiharibabu I have decided not to add more table statistics at this time relating to backup. I know MongoDB never collects table stats other than the oplog, which is not the likely interesting table for backup. This is ready for review.

File: test/suite/test_backup18.py [hunk 1]
```diff
@@ -68,6 +75,10 @@ def test_backup18(self):
         # Open up the backup cursor.
         config = 'incremental=(enabled,this_id="ID1")'
         bkup_c = self.session.open_cursor('backup:', None, config)
+        backup = self.get_stat(stat.conn.backup_cursor_open)
+        self.assertEqual(backup, 1)
+        backup = self.get_stat(stat.conn.backup_incremental)
+        self.assertEqual(backup, 1)
```
Comment (ddanderson) [inline test/suite/test_backup18.py:81]: This is not so readable when all values are assigned to `backup`.  Maybe pick names like `backup_cursor_open = self.get_stat(....)`

Or could do:
```
    self.assertEqual(1, self.get_stat(stat.conn.backup_cursor_open))
```

**Review by ddanderson — APPROVED** [review]
Makes sense and looks good, added one thought about readability.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**